### DISCUSSION TITLE
Feature Request: Drag and drop in mixer

### DISCUSSION THREAD

**Discussion by Lukajnk** [File-New-Project/EarTrumpet#1006] [Ideas]
First I want to thank you for doing an amazing job. The app has made switching output devices for each program so much easier which used to be such a hassle especially when you have to do it often when streaming and using a virtual audio device.

When the volume mixer is open (the right click > volume mixer one) and when you expand the menu by clicking on the main output device all the devices and programs are visible. Since all programs and devices are visible and this app is all about changing settings faster I thought it would be a good idea to implement drag-and-drop functionality where you can simply drag a program from audio device A to audio device B. Restoring a program's output device to the default output device could be done by right-clicking on the programing and doing so.

I don't know if someone already submitted this feature request nor do I have the knowledge about if it's possible to implement or if it creates more problems than it solves, but it came to mind as something that wouldn't surprise me the program had by default as it feels intuitive.

This is my first time writing a feature request or any post on GitHub so apologies if I don't follow a posting guideline.

**Comment by thrynae** [comment]
I just started using it, and this is one of the very few things that is easier with the Windows utility: switching your default output.
I might have just missed it, but I wholeheartedly support this idea.

**Comment by theiwaz** [comment]
see #662
